db/seedfiles/translations/en/cca/employer.rb

```diff
@@ -19,10 +19,10 @@
   :'en.employers.plan_years.benefit_package.contributions.employer_contribution' => "You'll need to contribute at least %{aca_shop_market_employer_contribution_percent_minimum}% toward the cost of monthly premiums based on the Reference Plan or One Plan you chose. If you decide to offer family coverage, you will need to contribute at least %{aca_shop_market_employer_family_contribution_percent_minimum}% toward family premiums. Note: We have a special annual enrollment period for group plans that start in January. If you enroll during the special annual enrollment period, these contribution requirements do not apply.",
   :'en.employers.plan_years.benefit_package.employee_cost_button' => "Employee Detailed Costs",
   :'en.employers.employer_profiles.my_account.important_documents_needed' => "Important Documents Needed",
-  :'en.employers.employer_profiles.my_account.eligibility_verification_html' => "You will need to submit proof that your group is an eligible Massachusetts small business before you can complete the enrollment process.</br></br>Please review our list of <a href='%{link_to}' target='_blank'>documents that can be used as proof</a> and upload PDF document(s) that meet the requirements.</br></br>After you’ve uploaded your documents, you can continue with choosing your benefit package. We will review your documents and let you know if we need any further documentation.",
+  :'en.employers.employer_profiles.my_account.eligibility_verification_html' => "You will need to submit proof that your group is an eligible Maine small business before you can complete the enrollment process.</br></br>Please review our list of <a href='%{link_to}' target='_blank'>documents that can be used as proof</a> and upload PDF document(s) that meet the requirements.</br></br>After you’ve uploaded your documents, you can continue with choosing your benefit package. We will review your documents and let you know if we need any further documentation.",
```

Comment:
I think what we need to do here is just take the original code from `disable_shop_base` designed for Massachusetts and merge that into `me_demo`. Anytime you see `cca` that's the `site_key` for Massachusetts. And it looks like the line eligibility_verification_html is referring to employers stuff, which we don't have on Maine anyway. @eidarus